packages/teams-js/src/public/app.ts

```diff
@@ -727,12 +736,18 @@ export namespace app {
    *
    * @param validMessageOrigins - Optionally specify a list of cross frame message origins. They must have
    * https: protocol otherwise they will be ignored. Example: https://www.example.com
+   *
+   * @param customMessageIdStarter -
```

Comment:
Any help in explaining the parameter please?

Reply:
represents that first number in a monotonically increasing sequence of numbers we stamp the requests from the teamsJS to hubsdk. It helps in matching responses to requests.  
Having a high seed value for this field results in generating messageIds that have a less change of collision when there are multiple instances of teamsJS running by the application.